tests/cpp/dlr_test.cc

```diff
@@ -301,6 +302,44 @@ TEST(DLR, TestCreateFromPaths_RelayVM) {
   DeleteDLRModel(&model);
 }
 
+TEST(DLR, TestMakePathVecForSingleWindowsPath) {
+  std::string windows_path = "C:\\a\\b\\c:";
+  std::vector<std::string> path_vec = dlr::MakePathVec(windows_path);
+  EXPECT_EQ(path_vec.size(), 1);
+  EXPECT_EQ(path_vec[0], "C:\\a\\b\\c");
+}
+
+TEST(DLR, TestMakePathVecForMultipleWindowsPaths) {
+  std::string windows_path = "C:\\a\\b\\c:D:\\a\\b\\c";
+  std::vector<std::string> path_vec = dlr::MakePathVec(windows_path);
+  EXPECT_EQ(path_vec[0], "C:\\a\\b\\c");
+  EXPECT_EQ(path_vec[1], "D:\\a\\b\\c");
+}
+
+TEST(DLR, TestMakePathVecForSingleLinuxPath) {
+  std::string linux_path = "/a/b/c:";
```

Comment:
Can you add a test for single linux path without `:` at end?

Ex: `std::string linux_path = "/a/b/c";`